md/install-a-bonita-bpm-cluster.md

```diff
@@ -122,38 +119,36 @@ In this case you already have a Bonita BPM Platform running as single node insta
 
 ### Update the configuration in database
 
-Some properties of the Bonita BPM Platform needs to be changed in order to make the cluster work.
-
-1. Use your [platform setup tool](BonitaBPM_platform_setup.md) previously use to setup and configure your node. If you don't have a platform setup tool (which is unlikely), you can download the Bonita BPM [Deploy bundle](deploy-bundle.md) and unzip it at some place of your choice.
-::: info
-The platform setup tool is also present in the Tomcat or WildFly bundle under the `setup` directory.
-:::
-2. If the platform setup tool is not configured already, configure it as described in the [platform setup tool page](BonitaBPM_platform_setup.md). If it is, move on to step 3. 
-3. Run the `setup.sh pull` or `setup.bat pull`. This will retrieve the configuration of your platform under `platform_conf/current` folder.
+Some properties of the Bonita BPM Platform needs to be changed in order to make your installation work as a cluster node.
 
+1. Use your [platform setup tool](BonitaBPM_platform_setup.md) previously used to setup and configure your node. If you don't have a platform setup tool (which is unlikely), you can download the Bonita BPM [Deploy bundle](deploy-bundle.md) and unzip it at some place of your choice.
+    ::: info
+    The platform setup tool is also present in the Tomcat or WildFly bundle under the `setup` directory.
+    :::
+1. If the platform setup tool is not configured already, configure it as described in the [platform setup tool page](BonitaBPM_platform_setup.md). If it is, move on to step 3. 
```

Comment:
"move on to step 3. " may be moved to "move next step", since we don't want to bother about step ids